api/grant.go

```diff
@@ -35,13 +35,14 @@ func (r *CreateGrantResponse) StatusCode() int {
 }
 
 type ListGrantsRequest struct {
-	User          uid.ID `form:"user" note:"ID of user granted access" example:"6TjWTAgYYu"`
-	Group         uid.ID `form:"group" note:"ID of group granted access" example:"6k3Eqcqu6B"`
-	Resource      string `form:"resource" example:"production.namespace" note:"a resource name"`
-	Destination   string `form:"destination" example:"production" note:"name of the destination where a connector is installed"`
-	Privilege     string `form:"privilege" example:"view" note:"a role or permission"`
-	ShowInherited bool   `form:"showInherited" note:"if true, this field includes grants that the user inherits through groups" example:"true"`
-	ShowSystem    bool   `form:"showSystem" note:"if true, this shows the connector and other internal grants" example:"false"`
+	User             uid.ID `form:"user" note:"ID of user granted access" example:"6TjWTAgYYu"`
+	Group            uid.ID `form:"group" note:"ID of group granted access" example:"6k3Eqcqu6B"`
+	Resource         string `form:"resource" example:"production.namespace" note:"a resource name"`
+	Destination      string `form:"destination" example:"production" note:"name of the destination where a connector is installed"`
+	Privilege        string `form:"privilege" example:"view" note:"a role or permission"`
+	ShowInherited    bool   `form:"showInherited" note:"if true, this field includes grants that the user inherits through groups" example:"true"`
+	ExcludeConnector bool   `form:"-"`
```

Comment:
It feels like it may make more sense to just filter the connector out on the client-side rather than adding a flag for it specifically. What was the use-case here, the UI?

Reply:
It makes it easier to filter in the UI. This flag is just for backwards compatibility to the old API.